### ISSUE TITLE
Assigning materials as trans

### ISSUE THREAD

**Issue by sariths** [tbleicher/su2rad#23]
Hi @tbleicher , I have the 2014 version of your plugin (that you shared through a Dropbox link). I noticed that I could not get trans material to export despite changing opacity values. The relevant code is [here](https://github.com/tbleicher/su2rad/blob/4874befa4c1b7af5e95d6fc9b2da6c6f2027c7d5/su2rad/su2radlib/materialcontext.rb#L423-L433). I can get it to work if I obtain the alpha value from skm (sketchup material?) instead of color.alpha? I just wanted to check with you if there was a way to export to trans with the way your code is set up currently.

**Comment by tbleicher** [comment]
Hi Sarith.

I may have to update this part of the exporter to reflect changes how newer
versions of SketchUp structure the data. I think it used to work with
version 8 when I wrote the initial exporter. It is an evil hack anyway. The
proper way to do it is to predefine materials in a library ("ray"
subdirectory of the plugin) and assign the Radiance material to the
SketchUp material via drag-n-drop. The SketchUp material name will be
aliased to the correct Radiance definition which can be as complex as you
like.

Thank you for bringing this to my attention. I will update the code when I
get the chance.

Regards,
Thomas

**Comment by sariths** [comment]
Hi Thomas,


Thanks for the quick reply. I agree that predefining them through the ray
subdirectory is a better option. I have been using your plugin regularly
for the last few years and have actually replaced all your existing
materials in that directory with my own (because I found some of the names
to be too cryptic).

I use the automatic assign-by-color thing to setup a scene, view it through
objview etc. So it is actually quite useful for me. Anyways, although I
don't know any Ruby, the lines that I modified in your code actually do the
job for me as far as the trans thing is concerned. I look forward to
updates from you as when you get to it. Thanks for all the continuing work
on this plugin.

Regards,
Sarith
